lib/solargraph/source/chain/call.rb

```diff
@@ -197,7 +197,7 @@ def arguments_match arguments, signature
 
         # @param api_map [ApiMap]
         # @param name_pin [Pin::Base]
-        # @return [Array<Pin::Base>]
+        # @return [::Array<Pin::Base>]
```

Comment:
This clarifies between Solargraph::Chain::Array and the stdlib Array class.  I have another fix in the works to fix a bug where we confuse the two during typechecking and report a false positive, but this at least makes the declaration correct.